content/openapi/introduction.md

```diff
@@ -35,7 +35,12 @@ async function bootstrap() {
     .addTag('cats')
     .build();
   const document = SwaggerModule.createDocument(app, options);
-  SwaggerModule.setup('api', app, document);
+  SwaggerModule.setup('api', app, document, {
+    swaggerOptions: {
+      persistAuthorization: true,
+    },
+    customSiteTitle: 'My website browser title.',
+  });
```

Comment:
If we're going to add this, it should be explicitly stated as to what this configuration does, otherwise it's kind of just shooting in the dark for others and hoping it works.

Reply:
Since the object name was so obvious, I didn't have any idea where to mention it. But since `persistAuthorization` wasn't mentioned anywhere, It was very hard to test APIs.

Reply:
We shouldn't update the most basic example, but rather describe available configuration properties later in the doc.

Reply:
It's done. Please check it and let me know if there is more to do.

Reply:
> We shouldn't update the most basic example

Also removed the option from the main example.